### PR TITLE
JS3 week 2 homework submission.

### PR DESCRIPTION
**Pull request by shanhaider23**

### CODE REVIEW

File: javascript/javascript3/week2/excercise.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    setTimeout(() => {
+      resolve();
+    }, resolveAfter * 3000);
+  });
+}
+resolveTimeout(8).then(() => {
+  console.log("I am called asynchronously.");
+});
+
+async function resolveTimeout2(resolveAfter) {
```
Comment (Deepti-Dk) [inline javascript/javascript3/week2/excercise.js]: Function name could be changed to more logical name like resolveTimeoutWithAsync() instead on resolveTimeout2.

**Review by Deepti-Dk — APPROVED** [review]
clean and precise code
logical variable and function names
working perfect
well done :-)

File: javascript/javascript3/week2/excercise.js (hunk 1 above)
Comment (shanhaider23) [reply javascript/javascript3/week2/excercise.js]: Thanks for your suggestion. I will change it

**Comment by shanhaider23** [comment]
thanks for comments

**Pull request merged**